packages/atlas/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: atlas
 description: An extensible map abstraction for Flutter with support for multiple map providers
-version: 0.5.0
+version: 0.6.0
```

Comment:
Since this is a non-breaking change we can just bump to 0.5.1

Reply:
Changes committed.  Thanks!